csaf_2.1/prose/edit/src/schema-elements-02-props-03-vulnerabilities.md

```diff
@@ -370,6 +370,14 @@ Vulnerability notes (`notes`) of value type Notes Type (`notes_t`) holds notes a
     },
 ```
 
+The following combinations of `category` and `title` have a special meaning and MUST be used as stated below:
+
+| `category` | `title` | content of `text` |
```

Comment:
Why "content of `text`" instead of simply `text`?

Reply:
For `category` and `title` we provide the exact wording that has to be used. The `text` is a description of the content it should have...
I'm okay with changing it, if that argument doesn't convince you / the TC.